lib/parseTriggers.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var FirebaseError = require('./error');
+var fork = require('child_process').fork;
+var path = require('path');
+var RSVP = require('rsvp');
+
+var TRIGGER_PARSER = path.resolve(__dirname, './triggerParser.js');
+
+module.exports = function(projectId, sourceDir) {
```

Comment:
Also, this file uses FirebaseErrors, whereas the emulator trigger parser uses plain JS errors. This is because when a FirebaseError is passed into the reject function, the full message doesn't show up here: https://github.com/firebase/firebase-tools/blob/functions-emulator/lib/serve/functions.js#L102